pipelines/buildcallmanual.yml

```diff
@@ -0,0 +1,68 @@
+# Pipeline for creating files and running tests for Build Call.
+
+ variables:
+- template: config/settings.yml
+- name: packagingEnabled
+  value: false
+
+parameters:
+- name: build
+  displayName: Build binary selection
+  default: common
+  values:
+  - all
+  - common
+  
+trigger: none
+
+jobs:
+- job: BuildCallUnity2019Setup
+  displayName: Build Call Unity 2019 binaries and manual testing setup
+  timeoutInMinutes: 120
+  pool:
+    name: On-Prem Unity
+    demands:
+    - Unity2019.4.8f1
+    - COG-UnityCache-WUS2-01
+    - SDK_18362 -equals TRUE
+  steps:
+  - ${{ if contains(parameters.build, 'all') }}:
+  - template: templates/buildcallsetup.yml
```

Comment:
you _might_ need to indent things underneath the - ${{if statements

Reply:
I believe that I have fixed the indentation with the last commit.